src/ResourceManager/AnalysisServices/Commands.AnalysisServices.ServiceManagement/Commands/Restart-AzureAsInstance.cs

```diff
@@ -0,0 +1,149 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Instrumentation;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.AnalysisServices.ServiceManagement.Properties;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+namespace Microsoft.Azure.Commands.AnalysisServices.ServiceManagement
+{
+    /// <summary>
+    /// Cmdlet to log into an Analysis Services environment
+    /// </summary>
+    [Cmdlet("Restart", "AzureAsInstance", SupportsShouldProcess=true)]
+    [Alias("Restart-AzureAs")]
+    [OutputType(typeof(AsAzureProfile))]
+    public class RestartAzureAnalysisServer: AzurePSCmdlet, IModuleAssemblyInitializer
```

Comment:
>RestartAzureAnalysisServer [](start = 17, length = 26)

please follow the comments in the respective RP directory ChangeLog.md and submit your changelog
src/ResourceManager/<Service>/ChangeLog.md